packages/forma-36-react-components/src/components/Button/Button.tsx

```diff
@@ -23,7 +23,7 @@ export type ButtonProps = {
   loading?: boolean;
   disabled?: boolean;
   testId?: string;
-  buttonType?: 'primary' | 'positive' | 'negative' | 'muted' | 'naked';
+  buttonType?: 'primary' | 'positive' | 'negative' | 'warning' | 'muted' | 'naked';
```

Comment:
Is `warning` even valid? Does it actually have a different styling?

Reply:
it is, it exists in the story file and the css :)

Reply:
Hey @davidfateh , yeah, seems like it's a valid case and has different styling https://f36-storybook.contentful.com/?path=/story/components-button--button